docs/proposals/kubeedgeadm-scope.md

```diff
@@ -77,13 +77,13 @@ Examples:
     ┌──────────────────────────────────────────────────────────┐
     │ On the first machine:                                    │
     ├──────────────────────────────────────────────────────────┤
-    │ cloud-node#  kubeedge init <options>                 │
+    │ cloud-node#  keadm init <options>                        │
     └──────────────────────────────────────────────────────────┘
 
     ┌──────────────────────────────────────────────────────────┐
     │ On the second machine:                                   │
     ├──────────────────────────────────────────────────────────┤
-    │ edge-node#  kubeedge join <options>                  │
+    │ edge-node#  keadm join <options>                         │
```

Comment:
The worker node (at the edge)#  keadm join <options>

Reply:
Thanks. Done